cellprofiler/modules/trackobjects.py

```diff
@@ -1032,6 +1104,52 @@ def draw(self, labels, ax, object_numbers):
                 ax.annotate(str(n), xy=(x, y), color='white',
                             arrowprops=dict(visible=False))
 
+    def run_followneighbors(self, workspace, objects):
+        '''Track objects based on following neighbors'''
+        def calculate_iteration_value(param, initial_value):
+            iteration_default = NeighbourMovementTrackingParameters.parameters_cost_iteration[param]
+            initial_default = NeighbourMovementTrackingParameters.parameters_cost_initial[param]
+            return float(iteration_default) / initial_default * initial_value
+
+        tracker = NeighbourMovementTracking()
+        tracker.parameters_tracking["avgCellDiameter"] = self.average_cell_diameter.value
+        tracker.parameters_tracking["max_distance"] = self.pixel_radius.value
+
+        tracker.parameters_cost_initial["default_empty_cost"] = self.drop_cost.value
+        tracker.parameters_cost_iteration["default_empty_cost"] = calculate_iteration_value("default_empty_cost", self.drop_cost.value)
+
+        tracker.parameters_cost_initial["area_weight"] = self.area_weight.value
+        tracker.parameters_cost_iteration["area_weight"] = calculate_iteration_value("area_weight", self.area_weight.value)
+
+        old_labels = self.get_saved_labels(workspace)
+        if not old_labels is None:
```

Comment:
`if not/else` can be more simply written as an `if/else` by swapping the encapsulated code blocks.